R/Filtering.R

```diff
@@ -502,3 +502,251 @@ FilterParFieldReview <- function(field, .data, parameter) {
   print(pie)
   print(df)
 }
+
+#' Unique Field Values Table
+#'
+#' Function creates a table yielding the number of unique values in each field (column) returned.
+#' The first column of the summary table holds the field name and the second column holds the
+#' unique value count.
+#' 
+#' @param .data TADA dataframe
+#' @param display A character string denoting what fields to return in the summary table. Defaults to "key". "all" will return all fields in the dataset, "narrow" will return most field names except those holding numeric values or units, and "key" returns the most important columns to review. Note that if a field is completely NA, it will not be shown on the summary table.
```

Comment:
Suggestion (use or ignore). 

narrow and key are synonyms in my mind because we use narrow for the WQP profiles to indicate it includes only the key elements. Not sure if that will confuse others or if it's just me. What do you think of using "most" instead of narrow? Or "priority" instead of key.

Reply:
changed from "narrow" to "most"